internal/types/document.go

```diff
@@ -306,6 +306,16 @@ func (d *Document) RemoveByPath(path Path) {
 	removeByPath(d, path)
 }
 
+// SetByPath sets value by given path.
+func (d *Document) SetByPath(path Path, value any) error {
+	return setByPath(d, path, value)
+}
+
+// InsertByPath creates Document by given path if it's not exist and sets given value to that Document.
+func (d *Document) InsertByPath(path Path, value any) error {
```

Comment:
If it's always a `*types.Document`, let's set the variable name to `*types.Document`, not `any`.
Otherwise, let's change the comment?

Reply:
Rewrote the comment